app/screens/scheduled_post_options/footer/scheduled_post_footer.tsx

```diff
@@ -68,13 +95,32 @@ export function ScheduledPostFooter({onSchedule, ...props}: Props) {
         >
             <TouchableOpacity
                 onPress={onSchedule}
-                style={style.applyButton}
+                style={isScheduling ? style.disabledApplyButton : style.applyButton}
+                disabled={isScheduling}
             >
-                <FormattedText
-                    id='scheduled_post_options.confirm_button.text'
-                    defaultMessage='Schedule Draft'
-                    style={style.applyButtonText}
-                />
+                {
+                    isScheduling &&
+                    (
+                        <View style={style.schedulingButtonContentWrapper}>
+                            <View style={{position: 'relative', height: '100%', width: 20}}>
+                                <Updating/>
+                            </View>
+                            <FormattedText
+                                id='scheduled_post_options.confirm_button.processing.text'
+                                defaultMessage='Scheduling'
+                                style={style.applyButtonProcessingText}
+                            />
+                        </View>
+                    )
+                }
+                {
+                    !isScheduling &&
+                    <FormattedText
+                        id='scheduled_post_options.confirm_button.text'
+                        defaultMessage='Schedule Draft'
+                        style={style.applyButtonText}
+                    />
+                }
             </TouchableOpacity>
```

Comment:
Is this a button? Should we use the `<Button>` component implemented in `components/button`?

Reply:
Button component is causing too many styling issues and I'll need to modify that component to allow making it look like the designs and then I'll have to verify its working finer everywhere in the app so using this for now. Does this cause any problem BTW? Trying to understand this as we're using TouchableOpacity as button somewhere.

Filed https://mattermost.atlassian.net/browse/MM-62757 to track this.